### ISSUE TITLE
RouteData dictionary can contain null values

### ISSUE THREAD

**Issue by richardversluis** [mroach/RollbarSharp#34]
The Dictionary RouteData can contain values which are null. RollbarSharp doesn't take this into account and is doing a `value.ToString()` which results in a (swallowed) exception. Line with the bug:

```c#
m.Parameters = request.RequestContext.RouteData.Values.ToDictionary(v => v.Key, v => v.Value.ToString());
````

https://github.com/mroach/RollbarSharp/blob/master/src/RollbarSharp/Builders/RequestModelBuilder.cs#L53

**Comment by mroach** [comment]
Replacing `.ToString()` with [.Describe()](https://github.com/mroach/RollbarSharp/blob/master/src/RollbarSharp/InternalExtensions.cs#L72) would take care of it.

**Comment by AlbertoMonteiro** [comment]
@mroach do you still mataining this package?

**Comment by mroach** [comment]
@AlbertoMonteiro Not really. I've mostly moved over to Ruby land these days. @webcoyote is a maintainer and I thought I had made @richardversluis one, but I haven't. Perhaps he'd be interested? ;)

**Comment by AlbertoMonteiro** [comment]
@mroach I hope, I am using Rollbar now and should be good to have this library updated.
Btw, why did you moved to Ruby?

**Comment by mroach** [comment]
If one of the other guys can't get to it in the next couple days I'll see if my Windows VM can still build, test, and deploy updates without too much effort. I don't like having Windows in my life or products that require licensing. Sublime Text being the one tolerable exception to that :)

**Comment by AlbertoMonteiro** [comment]
@mroach I would like to be a maintainer.

Did you see that next version of asp.net will be cross-platform and will run on any text editor, like sublime?

**Comment by richardversluis** [comment]
@mroach I'm interested.

**Comment by mroach** [comment]
@richardversluis @AlbertoMonteiro Welcome to the collaborator-o-sphere

**Comment by AlbertoMonteiro** [comment]
Thank you very much @mroach

**Comment by AlbertoMonteiro** [comment]
@mroach could you add we to nuget maintainers too?

**Comment by richardversluis** [comment]
fixed in e496916b586aee69c38aa626437aab05a1618956
